.gitignore

```diff
@@ -16,6 +16,7 @@ cache.properties
 !var/sessions/.gitkeep
 /var/bootstrap.php.cache
 !var/SymfonyRequirements.php
-/build/
+/web/build
```

Comment:
Personally, i like to add a `/` at the end here to indicate that this is a directory. What do you think?